arch/x86/x86_features.c

```diff
@@ -15,6 +15,13 @@
 #else
 // Newer versions of GCC and clang come with cpuid.h
 #  include <cpuid.h>
+#  ifdef X86_HAVE_XSAVE_INTRIN
+#    if __GNUC__ == 8
```

Comment:
So this is only needed for 8.x, right? Not 9.x and newer or 7.x and older?

Also, this place you only check for 8.x, the other places you check that minor is 1 or more as well.

Intentional?

Reply:
``X86_HAVE_XSAVE_INTRIN`` is not defined is gcc < 8

Reply:
> ``X86_HAVE_XSAVE_INTRIN`` is not defined is gcc < 8

It's not defined for gcc < 8.2 as the implementation is buggy...

Reply:
@Dead2 AFAIK, Clang sets `__GNUC__` to `4`, Intel icc/icpc sets it to `6` or `7` depending on what operating system is used (`6` for Ubuntu 16.04, `7` for Ubuntu 18.04, essentially to same as default gcc/g++ version on the specific distribution)